### PR TITLE
clickhouse_cfg_info: add the new module

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
clickhouse_cfg_info: add the new module

this is MVP

If it's OK, let's merge this one and then discuss future possibilities in a new issue

### CODE REVIEW

**Comment by Andersson007** [comment]
@aleksvagachev ready for review

File: plugins/modules/clickhouse_cfg_info.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+requirements:
+  - pyyaml
+'''
+
+# TODO: add the "dir" option
+# mutually exclusive with path.
+# If specified, it'll search the cfg files
+# in that directory and in config.d directory there.
+# Let's discuss it first
+
```
Comment (Andersson007) [inline plugins/modules/clickhouse_cfg_info.py:45]:
```suggestion
```

File: plugins/modules/clickhouse_cfg_info.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: clickhouse_cfg_info
+
+short_description: Retrieves ClickHouse config file content and returns it as JSON
+
```
Comment (Andersson007) [inline plugins/modules/clickhouse_cfg_info.py:15]:
```suggestion

version_added: '0.7.0'

```

**Comment by Andersson007** [comment]
ready for review

**Comment by aleksvagachev** [comment]
@Andersson007 Thanks for developing the new module!
1) Are you planning to add xml in the future?
2) Do we need to add a file existence check?
3) This module must be executed on a clickhouse server. This is not necessary for other modules. They are executed on one host, and the connection and execution of sql is on the other. Should we stick to such a system or not?

**Comment by Andersson007** [comment]
> @Andersson007 Thanks for developing the new module!
> 
> 1. Are you planning to add xml in the future?
> 2. Do we need to add a file existence check?
> 3. This module must be executed on a clickhouse server. This is not necessary for other modules. They are executed on one host, and the connection and execution of sql is on the other. Should we stick to such a system or not?

@aleksvagachev thanks for asking:
1. Yes, but would be happy to delegate it if you or anybody else who wants to become a co-author. I've already googled and it should be easy to implement with a couple of libs
2. Maybe, but not in this PR. A bit of more graceful error handling, would probably make sense
3. I believe no, it's OK

**Review by aleksvagachev — APPROVED** [review]

**Comment by Andersson007** [comment]
@aleksvagachev thanks for reviewing!
would you like to add xml support and the file error handling feature yourself? I could do it if not, no problem. Please let me know

**Comment by aleksvagachev** [comment]
@Andersson007 If you have free time, please implement it. If not, I can do it.

**Pull request merged**